CODEOWNERS

```
@@ -0,0 +1,3 @@
+# Community managmenent owners
+# When PR is open sig-community will be assigned and then individual maintainers
+*  @projectcontour/sig-community
```

should we maybe also add maintainers to the `sig-community` group and/or add to the change like below (just so we can use the github approver mechanisms automatically)

```suggestion
*  @projectcontour/sig-community @projectcontour/maintainers
```